articles/user-guide/host-programs.md

```diff
@@ -341,6 +341,68 @@ Multiple qubits:
 {'CNOT': 0, 'QubitClifford': 4, 'R': 0, 'Measure': 4, 'T': 0, 'Depth': 0, 'Width': 4, 'BorrowedWidth': 0}
 ```
 
+#### Using Q# code from other projects or packages
+
+By default, the `import qsharp` command loads all of the `.qs` files in the current folder and makes their Q# operations and functions
+available for use from inside the Python script.
+
+To load Q# code from another folder, the [`qsharp.projects` API](https://docs.microsoft.com/python/qsharp/qsharp.projects.projects)
+can be used to add a reference to a `.csproj` file for a Q# project (that is, a project that references `Microsoft.Quantum.Sdk`).
+This command will compile any `.qs` files in the folder containing the `.csproj` (and subfolders). It will also recursively load
+any packages referenced via `PackageReference` or Q# projects referenced via `ProjectReference` in that `.csproj` file.
+
+As an example, the following Python code imports an external project, referencing its path relative to the current folder,
+and invokes one of its Q# operations:
+
+```python
+import qsharp
+qsharp.projects.add("../qrng/Qrng.csproj")
+from Qrng import SampleQuantumRandomNumberGenerator
+print(f"Qrng result: {SampleQuantumRandomNumberGenerator.simulate()}")
+```
+
+This results in output like the following:
+
+```output
+Adding reference to project: ../qrng/Qrng.csproj
+Qrng result: 0
+```
+
+To load external packages containing Q# code, use the [`qsharp.packages` API](https://docs.microsoft.com/python/qsharp/qsharp.packages.packages).
+
+If the Q# code in the current folder depends on external projects or packages, you may see errors when running `import qsharp`,
+since the dependencies have not yet been loaded.
+To load required external packages or Q# projects during the `import qsharp` command, ensure that the folder with the Python script
+contains a `.csproj` file which references `Microsoft.Quantum.Sdk`. In that `.csproj`, add the property
+`<IQSharpLoadAutomatically>true</IQSharpLoadAutomatically>` to the `<PropertyGroup>`. This will instruct IQ# to recursively
```

Comment:
Is this still correct for this release, or was that part of what's deferred to next release?

Reply:
Yes, this is still correct, thanks for double-checking. The feature that we deferred to next release was just the language server change to generate a temporary .csproj file (and thereby enable IntelliSense for standalone .qs files). Nothing in IQ# is affected.

Reply:
(I see now that I may have created confusion by removing the wrong line from the release notes PR #1019. I've corrected that mistake.)